### PR TITLE
docs: add Vue Mastery links for Vue Router Cheat Sheet

### PR DESCRIPTION
**Pull request by tutorialcode**

### CODE REVIEW

File: packages/docs/.vitepress/components/VueMasteryHomeLink.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <div class="container">
+    <div class="inside">
+      <a href="https://www.vuemastery.com/vue-router/?coupon=ROUTER-DOCS&via=eduardo" target="_blank">
+        <span class="logo-wrapper">
+          <img alt="Vue Mastery Logo" width="25px" src="https://storage.googleapis.com/vue-mastery.appspot.com/flamelink/media/vue-mastery-logo.png">
```
[the remark this answers is not in the document]
Comment (tutorialcode) [reply packages/docs/.vitepress/components/VueMasteryHomeLink.vue]: Yeah, it's updated with a smaller image (50px)

**Comment by tutorialcode** [comment]
@posva I updated the image link with a smaller version (50px)

File: packages/docs/.vitepress/components/VueMasteryLogoLink.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+const props = defineProps<{
+  for: string,
```
Comment (posva) [inline packages/docs/.vitepress/components/VueMasteryLogoLink.vue:3]: let's just use a boolean
```suggestion
  description?: boolean,
```

File: packages/docs/.vitepress/components/VueMasteryLogoLink.vue [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+const forVueRouterCheatSheet = props.for === 'vue-router-cheat-sheet'
+</script>
+
+<template>
+  <a :href="link" target="_blank">
+    <span class="logo-wrapper">
+      <img alt="Vue Mastery Logo" width="25px" src="https://firebasestorage.googleapis.com/v0/b/vue-mastery.appspot.com/o/flamelink%2Fmedia%2Fvue-mastery-logo-small.png?alt=media&token=941fcc3a-2b6f-40e9-b4c8-56b3890da108">
+    </span>
+    <span v-if="forVueRouterCheatSheet" class="description">
```
Comment (posva) [inline packages/docs/.vitepress/components/VueMasteryLogoLink.vue:20]: that way, this becomes 
```suggestion
    <span v-if="description" class="description">
```

File: packages/docs/.vitepress/components/VueMasteryLogoLink.vue [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  'vue-router-cheat-sheet': 'https://www.vuemastery.com/vue-router/?coupon=ROUTER-DOCS&via=eduardo'
+}
+
+const link = links[props.for]
+
+const forVueRouterCheatSheet = props.for === 'vue-router-cheat-sheet'
+</script>
+
+<template>
+  <a :href="link" target="_blank">
```
Comment (posva) [inline packages/docs/.vitepress/components/VueMasteryLogoLink.vue:16]: Let's put the href directly here

File: packages/docs/.vitepress/components/VueMasteryLogoLink.vue [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  for: string,
+}>()
+
+const links = {
+  'vue-router-cheat-sheet': 'https://www.vuemastery.com/vue-router/?coupon=ROUTER-DOCS&via=eduardo'
+}
+
+const link = links[props.for]
+
+const forVueRouterCheatSheet = props.for === 'vue-router-cheat-sheet'
```
Comment (posva) [inline packages/docs/.vitepress/components/VueMasteryLogoLink.vue:12]: let's remove all of this and just put the href

**Review by posva — CHANGES_REQUESTED** [review]

**Comment by tutorialcode** [comment]
I refactored the code (putting the link in href directly)

**Review by posva — APPROVED** [review]

**Pull request merged**